api/all.yaml

```
@@ -1114,6 +1117,21 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/PermissionsList'
+  /metastore_summary:
+    get:
+      tags:
+        - Metastores
+      operationId: summary
+      summary: Get a metastore summary
+      description: |
+        Gets information about a metastore.
```

The request doesn't specify the metastore to fetch info for because currently the service has only one metastore. Make this clear in the description.

```suggestion
        Gets information about the metastore hosted by this Unity Catalog service (currently the service hosts only one metastore)
```